lib/internal/fs/streams.js

```diff
@@ -350,6 +362,15 @@ function WriteStream(path, options) {
     validateFunction(this[kFs].close, 'options.fs.close');
   }
 
+  this.flush = options.flush;
+  if (this.flush == undefined) {
```

Comment:
Any reason why checking for undefined but not null?

```suggestion
  if (this.flush == null) {
```